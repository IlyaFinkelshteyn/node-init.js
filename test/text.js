'use strict';

const test = require('ava');

const { withLine } = require('../lib/text.js');

test('withLine()', t => {
  const text = `
checkexisting
  checkwhitespace
check-hyphen
`;

  t.snapshot(withLine(text, 'checkexisting'));
  t.snapshot(withLine(text, 'checkwhitespace'));
  t.snapshot(withLine(text, 'check-hyphen'));
  t.snapshot(withLine(text, 'checkmissing'));
});
