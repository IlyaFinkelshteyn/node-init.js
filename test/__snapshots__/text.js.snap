// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`withLine() 1`] = `
"
checkexisting
  checkwhitespace
check-hyphen
"
`;

exports[`withLine() 2`] = `
"
checkexisting
  checkwhitespace
check-hyphen
"
`;

exports[`withLine() 3`] = `
"
checkexisting
  checkwhitespace
check-hyphen
"
`;

exports[`withLine() 4`] = `
"
checkexisting
  checkwhitespace
check-hyphen

checkmissing
"
`;
